components/raftstore-v2/src/raft/apply.rs

```diff
@@ -106,6 +130,16 @@ impl<EK: KvEngine, R> Apply<EK, R> {
         &mut self.region_state
     }
 
+    #[inline]
+    pub fn set_region_state(&mut self, region_state: RegionLocalState) {
```

Comment:
Seems not used.

Reply:
It's not needed now.

Reply:
Yes,  it is not needed now. I removed it.